pkg/driver/driver_alicloud.go

```diff
@@ -194,7 +129,7 @@ func (c *AlicloudDriver) Delete() error {
 		return err
 	} else if len(result) == 0 {
 		// No running instance exists with the given machineID
-		glog.V(2).Infof("No VM matching the machineID found on the provider %q", c.MachineID)
+		return fmt.Errorf("No VM matching the machineID found on the provider %s", c.MachineID)
```

Comment:
Can we revert this back to glog message, as other drivers use the same and we should maintain that consistency.